### ISSUE TITLE
Add global configuration options

### ISSUE THREAD

**Issue by lymanepp** [dolezsa/thermal_comfort#123]
I have quite a few temperature and humidity sensors. It would be nice to have global YAML configuration with overrides for common settings:
* Polling enabled
* Scan interval
* Sensor types

```
thermal_comfort:
  - sensor:
      - name: Garage
        temperature_sensor: sensor.garage_temperature
        humidity_sensor: sensor.garage_humidity
        unique_id: garage
        poll: true
        scan_interval: 60
        sensor_types:
          - absolute_humidity
          - dew_point
          - simmer_index

      - name: Guest Bath
        temperature_sensor: sensor.guest_bath_temperature
        humidity_sensor: sensor.guest_bath_humidity
        unique_id: guest_bath
        poll: true
        scan_interval: 60
        sensor_types:
          - absolute_humidity
          - dew_point
          - simmer_index

      - name: Hall
        temperature_sensor: sensor.hall_temperature
        humidity_sensor: sensor.hall_humidity
        unique_id: hall
        poll: true
        scan_interval: 60
        sensor_types:
          - absolute_humidity
          - dew_point
          - simmer_index

      - name: Lanai
        temperature_sensor: sensor.lanai_temperature
        humidity_sensor: sensor.lanai_humidity
        unique_id: lanai
        poll: true
        scan_interval: 60
        sensor_types:
          - absolute_humidity
          - dew_point
          - frost_point
          - frost_risk
          - simmer_index

      - name: Laundry
        temperature_sensor: sensor.laundry_temperature
        humidity_sensor: sensor.laundry_humidity
        unique_id: laundry
        poll: true
        scan_interval: 60
        sensor_types:
          - absolute_humidity
          - dew_point
          - simmer_index

      - name: Master Bath
        temperature_sensor: sensor.master_bath_temperature
        humidity_sensor: sensor.master_bath_humidity
        unique_id: master_bath
        poll: true
        scan_interval: 60
        sensor_types:
          - absolute_humidity
          - dew_point
          - simmer_index

      - name: Master Bed
        temperature_sensor: sensor.master_bed_temperature
        humidity_sensor: sensor.master_bed_humidity
        unique_id: master_bed
        poll: true
        scan_interval: 60
        sensor_types:
          - absolute_humidity
          - dew_point
          - simmer_index

      - name: Office
        temperature_sensor: sensor.office_temperature
        humidity_sensor: sensor.office_humidity
        unique_id: office
        poll: true
        scan_interval: 60
        sensor_types:
          - absolute_humidity
          - dew_point
          - simmer_index

      - name: Porch
        temperature_sensor: sensor.porch_temperature
        humidity_sensor: sensor.porch_humidity
        unique_id: porch
        poll: true
        scan_interval: 60
        sensor_types:
          - absolute_humidity
          - dew_point
          - frost_point
          - frost_risk
          - simmer_index
```

**Comment by lymanepp** [comment]
Looks good to me. Since sensor_types makes up much of my configuration, it would also be nice to have sensor_types work like that.

**Comment by rautesamtr** [comment]
I have an idea how this probably could be implemented nicely. We should also include `custom_icons`.

**Comment by rautesamtr** [comment]
Could you try #133? it should make something like this possible:

```yaml
thermal_comfort:
  - poll: true
    scan_interval: 60
    sensor_types:
      - absolute_humidity
      - dew_point
      - simmer_index

    sensor:
      - name: Garage
        temperature_sensor: sensor.garage_temperature
        humidity_sensor: sensor.garage_humidity
        unique_id: garage

      - name: Guest Bath
        temperature_sensor: sensor.guest_bath_temperature
        humidity_sensor: sensor.guest_bath_humidity
        unique_id: guest_bath

      - name: Hall
        temperature_sensor: sensor.hall_temperature
        humidity_sensor: sensor.hall_humidity
        unique_id: hall

      - name: Laundry
        temperature_sensor: sensor.laundry_temperature
        humidity_sensor: sensor.laundry_humidity
        unique_id: laundry

      - name: Master Bath
        temperature_sensor: sensor.master_bath_temperature
        humidity_sensor: sensor.master_bath_humidity
        unique_id: master_bath

      - name: Master Bed
        temperature_sensor: sensor.master_bed_temperature
        humidity_sensor: sensor.master_bed_humidity
        unique_id: master_bed

      - name: Office
        temperature_sensor: sensor.office_temperature
        humidity_sensor: sensor.office_humidity
        unique_id: office

  - poll: true
    scan_interval: 60
    sensor_types:
          - absolute_humidity
          - dew_point
          - frost_point
          - frost_risk
          - simmer_index

     sensor:
       - name: Porch
         temperature_sensor: sensor.porch_temperature
         humidity_sensor: sensor.porch_humidity
         unique_id: porch

       - name: Lanai
         temperature_sensor: sensor.lanai_temperature
         humidity_sensor: sensor.lanai_humidity
         unique_id: lanai
```

**Comment by lymanepp** [comment]
Other than a minor YAML format issue (extra space in the last 'sensor' section), this works great! The configuration is much nice with no duplicated values. Thanks for this!

**Comment by rautesamtr** [comment]
Closed with #133
